distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc

```diff
@@ -9,101 +9,101 @@
 |Example Value
 
 |contact.creator-name
-|The name of this metacard’s creator.
-|String
-|< 1024 characters
+|The name(s) of this metacard’s creator(s).
+|List of Strings
+|< 1024 characters per entry
 |
  
 |contact.creator-address
-|The physical address of this metacard’s creator.
-|String
-|< 1024 characters
+|The physical address(es) of this metacard’s creator(s).
```

Comment:
Correct, it is unconstrained. That way lies [madness](https://www.mjt.me.uk/posts/falsehoods-programmers-believe-about-addresses/).